File: Tarea 7/python/MaxMin.py

cantidad = 15
numeros = []
for i in range(cantidad):
    try:
        n = float(input(f"dame el numero {i+1}: "))
        numeros.append(n)
    except:
        print("SOlO NUMEROS, vuelve a intentarlo")
        exit()
max = numeros[0]
min = numeros [0]

for n in numeros:   
    if max<n:
        max = n
    if min>n:
        min = n
print("El maximo es: ", max)
print("El minimo es: ", min)

File: Tarea 7/c++/MAXMIN.cpp
#include <iostream>
using namespace std;
int main(){   
    int cantidad = 15;
    float numeros[cantidad];
    for(int i = 0; i<cantidad;i++){
        cout << "Dame el numero " << i+1 << endl;
        cin >> numeros[i];
    }
    float max = numeros[0];
    float min = numeros[0];

    for (int i = 0; i < cantidad; i++){
        if (max<numeros[i]){
            max = numeros[i];
        }
        if (min>numeros[i]){
            min = numeros[i];
        }   
    }
    cout << "El maximo es: " << max << endl;
    cout << "El minimo es: " << min << endl;

}